playbooks/analytics_sandbox.yml

```diff
@@ -19,13 +19,11 @@
   roles:
     - aws
     - security
-    - mysql
-    - edxlocal
-    - analytics_api
+    - common_vars
     - role: nginx
       nginx_sites:
         - insights
-        - analytics_api
```

Comment:
@pomegranited, this line was not present in https://github.com/open-craft/openedx-deployment/commit/cca980973364bf1136640e5649dd93f31de26296#diff-40c660c4b60b66b30b41740d0dd05ea3. So I have removed it.

Reply:
Sorry, I spoke too soon..  we do need that `analytics_api` line to exist under `nginx_sites`, and as a role on its own.  

epodx *do* expose their analytics API via nginx, though they use SSH tunnels to restrict access to the port itself.  I tested this by following the instructions in the secure repo docs for [accessing the Analytics API](https://github.com/hks-epod/epodx/blob/master/docs/api.md#hks-epodx---epodx-analytics-api).

Reply:
@pomegranited, I will fix this and redeploy insights after the LMS redeployment is done and everything is working. :+1: